3737/2/helpers/user_helper.rb

```diff
@@ -0,0 +1,10 @@
+require 'yaml'
+require_relative '../model/user_class'
+
+# class that check user
+class UserHelper
+  def self.exist_students_numbers?(enter_number)
```

Comment:
remember that variable names are nouns (ex. student_number)
also I think existed_student_number? better reflects what happens inside method